Tools/Scripts/webkitpy/api_tests/runner.py

```diff
@@ -146,10 +146,25 @@ def run(self, tests, num_workers):
 
             with TaskPool(
                 workers=self._num_workers,
```

Comment:
If `mutually_exclusive_groups` is defined, you want `_num_workers` to be half the number of cores by default, I think. Maybe the same as the number of cores.

Reply:
By absolute default `Tools/Scripts/webkitpy/port/base.py ` uses the same as number of cores. 
If I remove the default of `'--child-processes=1'` from `Tools/Scripts/run-api-tests` it will run in that fashion.